storm-core/test/jvm/org/apache/storm/TestCGroup.java

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Can we have the filenames also to be consistent, TestCgroup.java. Not a big deal though